jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,6 @@
+.level = INFO
```

Comment:
Currently they have to replace the file in a Custom image.  However, as we discussed on the call last week, we should look at some way of allowing a logging.properties file to be included in the application, to either be merged with this one or simply to replace it.